tests/tests.cpp

```diff
@@ -42,7 +42,7 @@
 #include <fstream>
 #include <string>
 
-#include "libzippp.h"
```

Comment:
Unfortunately, by changing that, you break the compilation within the Makefile. Can you fix it ?

Reply:
Sorry for that. But I wasn't able to compile without this change, using visual studio solution. By the way I don't know to fix, but could you what type of make system are you using?